### DISCUSSION TITLE
DirectX support in Linux

### DISCUSSION THREAD

**Discussion by chandrasekharps** [dotnet/Silk.NET#1872] [Q&A]
I am just trying to understand if there is any support for DirectX in native Linux (say Ubuntu 20.4)?
Tried a quad sample however getting an exception with context creation, (may be because of the window handle dependency) ?

**Comment by Beyley** [comment]
> I am just trying to understand if there is any support for DirectX in native Linux (say Ubuntu 20.4)? Tried a quad sample however getting an exception with context creation, (may be because of the window handle dependency) ?

We do support this through DXVK-native yes, you'll have to make sure to install the DXVK-native nuget package
